dev-libs/lager/lager-0.1.1-r1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Library to assist value-oriented design"
+HOMEPAGE="https://sinusoid.es/lager/"
+SRC_URI="https://github.com/arximboldi/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~riscv"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/boost:=
+	dev-libs/zug
+	dev-libs/immer
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	test? (
+		<dev-cpp/catch-3:0
+		dev-libs/cereal
+		dev-qt/qtcore
+		dev-qt/qtconcurrent
```

Comment:
Slots should be specified here. Looks like it only supports Qt 5. I'll fix on merging, thanks!